### PR TITLE
Update pinning button styling

### PR DESCRIPTION
**Pull request by sourcier**
🐿 v2.5.16

### CODE REVIEW

File: myft/main.scss [hunk 1]
```diff
@@ -196,11 +196,35 @@ $o-overlay-is-silent: true;
...
+	&[aria-pressed=true],
+	&:not([disabled]):hover {
+		background-color: transparent;
+		border: 0;
+	}
+}
+
+.myft-ui__pin-button {
+	$icon-width: 28;
+	@include oIconsGetIcon('pin', #66625f, $icon-width, $iconset-version: 1);
```
Comment (benbarnett) [inline myft/main.scss]: We should be using colours from the palette rather than hex colours directly

File: myft/main.scss [hunk 2]
```diff
@@ -196,11 +196,35 @@ $o-overlay-is-silent: true;
...
+	&[aria-pressed=true],
+	&:not([disabled]):hover {
+		@include oIconsGetIcon('pin', #990f3d, $icon-width, $iconset-version: 1);
+	}
+}
+
+.myft-ui__instant-alert-button {
+	$icon-width: 28;
+	@include oIconsGetIcon('mail', #66625f, $icon-width, $iconset-version: 1);
```
Comment (benbarnett) [inline myft/main.scss]: Same here re: colours

File: myft/templates/instant-alert.html [hunk 3]
```diff
@@ -9,12 +9,11 @@
...
 			<input type="hidden" value="http://www.ft.com/ontology/concept/Concept" name="directType">
 		{{/if}}
+		{{#if buttonText}}{{buttonText}}{{else}}Instant alerts off{{/if}}
 		<button
 			aria-label="Get instant alerts for {{name}}"
 			aria-pressed="false"
-			class="n-myft-ui__button
-				{{#variant}} n-myft-ui__button--{{this}}{{/variant}}
-				{{#size}} n-myft-ui__button--{{this}}{{/size}}"
+			class="myft-ui__instant-alert-button"
```
Comment (benbarnett) [inline myft/templates/instant-alert.html]: This button is used widely across the site – are you sure we can change the class name like this? (could be fine, just worth checking)

**Comment by sourcier** [comment]
screen shot 2018-02-06 at 16 40 39

**Comment by sourcier** [comment]
@benbarnett screenshots added.

**Pull request merged**